lib/spec_factory.hpp

```diff
@@ -78,7 +78,12 @@ class SpecFactory {
     //**** COMP ****/
     op_t random_op();
 
+    typedef WlSpec* (SpecFactory::*SpecGeneratorFuncPtr)();
+    static vector<SpecGeneratorFuncPtr> spec_generators;
+
 private:
+    static void initializeSpecs();
```

Comment:
I think now we can make it non-static 🤔

Reply:
[Update spec_factory.hpp](https://github.com/all-things-networking/fperf/pull/28/commits/f99d01efde44d782b7a3de03cf89638cfb24a3b7)